### ISSUE TITLE
An index for dataset repositories

### ISSUE THREAD

**Issue by katjabercic** [MathHubInfo/mhd#31]
https://datagit.org/about/

Would something like this be helpful?

**Comment by tkw1536** [comment]
Yes, but would require quite a bit of tweaking to work for our purpose. Also, if we have large (> 100 GB) datasets, I doubt Git / Git LFS is a good idea.
